content/en/logs/guide/enrichment-tables.md

```diff
@@ -23,15 +23,56 @@ Define new entities in Datadog like customer details, service names and informat
+{{% /tab %}}
+
+{{% tab "AWS S3 Upload" %}}
+
+Enrichment tables can automatically pull a csv file from an AWS S3 bucket to keep your data up to date. The integration will look for changes to the csv file in S3, and when the file is updated it will replace the Enrichment Table with the new data. This also enables API updating via the S3 API once you’ve configured the initial Enrichment Table.
+
+In order to update Enrichment Tables from S3, Datadog uses the IAM Role in your AWS account that you configured for your [AWS integration][1]. If you have not yet created that Role, [follow these steps][2] to do so. To allow that Role to update your Enrichment Tables, add the following permission statement to its IAM policies. Be sure to edit the bucket names to match your environment.
+
+
+```json
+{
+	"Statement": [
+		{
+			"Sid": "EnrichmentTablesS3",
+			"Effect": "Allow",
+			"Action": [
+				"s3:GetObject",
+				"s3:ListBucket"
+			],
+			"Resource": [
+				"arn:aws:s3:::<MY_BUCKET_NAME_1/*>",
+				"arn:aws:s3:::<MY_BUCKET_NAME_2>"
+			]
+		}
+	],
+	"Version": "2012-10-17"
+}
+```
+### Define The Table
+
+Create an Enrichment Table by clicking **New Enrichment Table +**, adding a name, selecting AWS S3, filling out all the fields, clicking import, and finally defining the primary key on which all lookups will happen. It may take a few seconds between clicking import and having the preview available for setting up your primary key in the schema.
+
+{{< img src="logs/guide/enrichment-tables/configure-s3-enrichment-table.png" alt="Create an Enrichment Table" style="width:100%;">}}
+
+[1]: https://app.datadoghq.com/account/settings#integrations/amazon-web-services
+[2]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#installation
+{{% /tab %}}
+{{< /tabs >}}
 
 This Enrichment Table can now be used to add additional attributes to logs with the [Lookup Processor][1].
 
 ## Modify an Enrichment Table
 
-In order to modify an existing Enrichment Table with new data, click the **Update Data +** button in the top right corner. The CSV that is selected will be upserted into the table, meaning that all existing rows with the same primary key will be updated, and all new rows will be added. Once the table is saved, the upserted rows will be processed asynchronously and will be updated in the preview. It may take up to 10 minutes for the updated rows to appear in affected logs.
+In order to modify an existing Enrichment Table with new data, select a table then click the **Update Data +** button in the top right corner. The CSV that is selected will be upserted into the table, meaning that all existing rows with the same primary key will be updated, and all new rows will be added. Once the table is saved, the upserted rows will be processed asynchronously and will be updated in the preview. It may take up to 10 minutes for the updated rows to appear in affected logs.
```

Comment:
```suggestion
To modify an existing enrichment table with new data, select a table then click **Update Data +** in the top right corner. The selected CSV is upserted into the table, meaning that all existing rows with the same primary key are updated, and all new rows are added. Once the table is saved, the upserted rows are processed asynchronously and updated in the preview. It may take up to 10 minutes for the updated rows to appear in affected logs.
```